recipes/server_install.rb

```diff
@@ -28,6 +28,8 @@
 
 case node['platform_family']
 when 'rhel'
+  yum_package 'which'
```

Comment:
Without this package the centos7 rundeck could not find a config file. Thats why the tests were failing for centos7